var/spack/repos/builtin/packages/ldak/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ldak(Package):
+    """LDAK is a software package for analyzing GWAS data"""
+
+    homepage = "http://dougspeed.com/ldak/"
+    url      = "http://dougspeed.com/wp-content/uploads/source.zip"
+
+    version('5.1', sha256='9a3fe2fafc7b68cc57a17748a64db66f76b13acbd5e9a538ede20a46447fcf4a')
+
+    variant('mkl', default=False, description='Use MKL')
+
+    depends_on('zlib')
+    depends_on('blas')
+    depends_on('lapack')
+    depends_on('intel-mkl', when='+mkl')
```

Comment:
```suggestion
    depends_on('mkl', when='+mkl')
```
Spack has a virtual provider `mkl` that is provided by both `intel-mkl` and `intel-parallel-studio+mkl`